### PR TITLE
Update ContactInfo component: 

### PR DESCRIPTION
**Pull request by studioo**
add form to backoffice

### CODE REVIEW

File: src/components/ContactInfo/ContactInfo.tsx [hunk 1]
```diff
@@ -11,27 +33,32 @@ const ContactInfo = (props: ContactInfoProps) => {
...
-              <p>MEDICON a.s. Antala Staška 1670/80 140 00 Praha 4</p>
+              {address && <p>{address}</p>}
             </div>
           </div>
           <div className={'grid contact-info__grid__element'}>
             <img src="/assets/medicon/images/contact-info-2.png" alt="phone number"/>
             <div>
-              <a href={'tel:+420 261 006 111'}>tel.: +420 261 006 111</a><br/>
-              <a href={'tel:+420 261 006 210'}>fax: +420 261 006 210</a>
+              {phones && phones.map((item, i) => (
```
Comment (EmilioHerreraSoukup) [inline src/components/ContactInfo/ContactInfo.tsx]: use for descriptive name for variable in mapping, if you  are iteraring phones then call it phone.

File: src/components/ContactInfo/ContactInfo.tsx [hunk 2]
```diff
@@ -11,27 +33,32 @@ const ContactInfo = (props: ContactInfoProps) => {
...
+                <a href={`${item.type}:${item.name}`} key={i}>{item.type}: {item.name}</a>
+              ))}
             </div>
           </div>
           <div className={'grid contact-info__grid__element'}>
             <img src="/assets/medicon/images/contact-info-3.png" alt="e-mail"/>
             <div>
-              <a href={'mailto:info@mediconas.cz'}>info@mediconas.cz</a>
+              {emails && emails.map((item, i) => (
+                <a href={`mailto:${item.name}}`} key={i}>{item.name}</a>
```
Comment (EmilioHerreraSoukup) [inline src/components/ContactInfo/ContactInfo.tsx]: use for descriptive name for variable in mapping, if you  are iteraring phones then call it phone.

File: src/components/ContactInfo/ContactInfo.tsx [hunk 3]
```diff
@@ -11,27 +33,32 @@ const ContactInfo = (props: ContactInfoProps) => {
...
+              ))}
+              
             </div>
           </div>
           <div className={'grid contact-info__grid__element'}>
             <img src="/assets/medicon/images/contact-info-4.png" alt="ICO and DIC"/>
             <div>
-              <p>IČO: 284 63 293</p>
-              <p>DIČ: CZ 284 63 293</p>
+              {additional && additional.map((item, i) => (
```
Comment (EmilioHerreraSoukup) [inline src/components/ContactInfo/ContactInfo.tsx]: use for descriptive name for variable in mapping, if you  are iteraring phones then call it phone.

File: src/components/ContactInfo/ContactInfo.tsx (hunk 2 above)
Comment (EmilioHerreraSoukup) [reply src/components/ContactInfo/ContactInfo.tsx]: I dont think there will be more thank two phones in this section, therefore array seems unnecessary , maybe ask Peter for this info.

File: src/components/ContactInfo/ContactInfo.tsx (hunk 3 above)
Comment (EmilioHerreraSoukup) [reply src/components/ContactInfo/ContactInfo.tsx]: Same here maybe there wont be more additional fields

Comment (studioo) [reply src/components/ContactInfo/ContactInfo.tsx]: Okay, I asked Peter about numbers of fields in this component and others too.

**Pull request merged**